VogtPayroll2/Program.cs

```diff
@@ -1,12 +1,73 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace VogtPayroll2
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            PayrollManager payrollManager = new PayrollManager();
+            List<Employee> employeeList = new List<Employee>();
+
+            employeeList = PrintMenu();
+
+            foreach (var employee in employeeList)
+            {
+                employee.NetPay = employee.GetNetPay();
+                employee.DisplayEmployeeInfo(employee);
+
+            }
+
+            employeeList = payrollManager.GetNetPayOfAllEmployees(employeeList);
+            
+            payrollManager.GetMinimumAndMaximumNetPay(employeeList);
+
+        }
+
+        public static List<Employee> PrintMenu()
+        {
+            PayrollManager payrollManager = new PayrollManager();
+            List<Employee> employeeList = new List<Employee>();
```

Comment:
there is no reason to create yet another employee list. you already have one above, which that one should be moved to PAyrollManager